pkg/apis/apps/v1/conversion_test.go

```diff
@@ -63,18 +63,20 @@ func TestV12StatefulSetSpecConversion(t *testing.T) {
 		},
 		"StatefulSetSpec Conversion 2": {
 			stsSpec1: &apps.StatefulSetSpec{
-				Replicas:            *replicas,
-				Selector:            selector,
-				Template:            apiTemplate,
-				ServiceName:         "foo",
-				PodManagementPolicy: apps.PodManagementPolicyType("bar"),
```

Comment:
nit: Would it be possible to split the commit into two commits - one for the functionality and one for the tests. This will help to wade through the commits in future.

Reply:
done